dashboard/urls.py

```diff
@@ -18,10 +18,12 @@
 from django.contrib.auth.decorators import login_required
 
 from django.conf import settings
-from django.conf.urls import include
+from django.conf.urls import include, url
```

Comment:
Might have missed this before when I was making the BE work with Django 2.2., but we shouldn't need `url` here anymore.